server/plugin.go

```diff
@@ -618,3 +628,12 @@ func (p *Plugin) checkAndRefreshToken(token *oauth2.Token) (*oauth2.Token, error
 
 	return nil, nil
 }
+
+func (p *Plugin) handleRevokedToken(info *gitlab.UserInfo) {
+	p.disconnectGitlabAccount(info.UserID)
+	err := p.CreateBotDMPost(info.UserID, "Your GitLab account was disconnected due to an invalid or revoked authorization token. Reconnect your account using the `/gitlab connect` command.", "custom_git_revoked_token")
```

Comment:
Since this isn't a custom post that the webapp plugin is going to render, we can leave the `postType` out of this function call:

```suggestion
	err := p.CreateBotDMPost(info.UserID, "Your GitLab account was disconnected due to an invalid or revoked authorization token. Reconnect your account using the `/gitlab connect` command.", "")
```

Reply:
OK. I will note that there are other DM posts which specify custom post types such as `custom_git_welcome`, `custom_git_todo`. But I don't see where the webapp contains definitions for any of these. I was trying to just make it follow that convention.

Reply:
1/5 consistency is key. If we ever want to add custom post rendering, the post are already marked.